gitlab/mixins.py

```diff
@@ -298,6 +298,9 @@ def save(self, **kwargs):
             GitlabUpdateError: If the server cannot perform the request
         """
         updated_data = self._get_updated_data()
+        # Nothing to update. Server fails if sent an empty dict.
+        if len(updated_data.keys()) == 0:
```

Comment:
Thanks for the update!

A more pythonic way to write this would be:
```python
if not update_data:
    return
```

Could you update the code? Thanks again.